frontend/apps/vara-man/src/feature/single-game/components/timer/index.tsx

```diff
@@ -2,40 +2,41 @@ import { useEffect, useState } from 'react';
 import { useAtom } from 'jotai';
 import { GAME_OVER } from '@/feature/game/consts';
 
-type Props = {
-  gameRestarted: boolean;
-};
+const totalTimeGame = 10 * 60 * 1000;
 
-export const GameTimer = ({ gameRestarted }: Props) => {
-  const [, setGameOver] = useAtom(GAME_OVER);
+export const GameTimer = () => {
+  const [gameOver, setGameOver] = useAtom(GAME_OVER);
+  const [remainingTime, setRemainingTime] = useState(totalTimeGame);
```

Comment:
Seems like a [redundant state](https://react.dev/learn/choosing-the-state-structure#avoid-redundant-state).

We can remove `timeLeft` state in this case and just calculate it from `remainingTime`

Reply:
@vraja-nayaka 

I need a little different logic here. I have to memorize the time when I have time left and when I lose, so that I can use this time when I play again. At the end of the timer I will restart it.